mk/generate.mk

```diff
@@ -109,6 +109,10 @@ generate/oas: $(GENERATE_OAS_PREREQUISITES)
 		PATH=$(CI_TOOLS_BIN_DIR):$$PATH oapi-codegen -config api/openapi/openapi.cfg.yaml -o api/openapi/types/$$(dirname $${DEST}})/zz_generated.$$(basename $${DEST}).go $${endpoint}.yaml; \
 	done
 
+.PHONY: generate/oas-for-ts
```

Comment:
should this be a dependency of some other target?

Reply:
This is what I'm using as my command `make generate/oas-for-ts`

Probably a bad name, and tbh I was tempted to change things a bit more, but I figured I'd be best not changing too much.